src/style.css

```
@@ -1 +1,81 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  margin: 0;
+}
+
+body {
+  background-color: #e5e5e5;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+}
+
+header {
+  height: 60px;
+  background: white;
+  width: auto;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: flex;
+  justify-content: right;
+  align-items: center;
+  flex-shrink: 1;
+  width: 250;
+  min-width: 100px;
+}
+
+.navigation {
+  margin-right: 3%;
+  flex-shrink: 0;
+}
+
+.nav_list {
+  padding: 0;
+  list-style: none;
+  right: 10%;
```

```suggestion
  right: 10%;
```
не варто тут викоритовувати відсоткові значення.
Має бути відступ 50 пікселів

---

Комент залишився без уваги